lib/Common/Exceptions/ReportError.cpp

```diff
@@ -19,6 +19,9 @@ void ReportFatalException(
     if (IsDebuggerPresent())
     {
         DebugBreak();
+#if DBG
+        return;
+#endif
```

Comment:
Curious what this change is for? Is it to enable continue execution in debugger after an assertion?